lib/livebook_web/live/output/plain_text_component.ex

```diff
@@ -42,7 +49,19 @@ defmodule LivebookWeb.Output.PlainTextComponent do
       phx-update="stream"
       phx-no-format
     ><span
-      :for={{dom_id, chunk}<- @streams.chunks} id={dom_id}>{chunk.text}</span></div>
+      :for={{dom_id, chunk}<- @streams.chunks} id={dom_id} style={format_style(chunk.style)}>{chunk.text}</span></div>
     """
   end
+
+  defp format_style(enum) do
+    Enum.map_join(enum, "; ", fn {key, value} when key in [:color, :font_weight, :font_size] ->
```

Comment:
Oh, we should actually ignore unknown keys!

If we support more styles in the future, there is a chance someone will use latest Kino, but old Livebook, so the Kino could allow `style: [background_color: "..."]` and we don't want Livebook to crash.